bootstrap/bootstrap.go

```diff
@@ -672,6 +674,19 @@ func (b *Bootstrap) executePluginHook(name string, checkouts []*pluginCheckout)
 	hookTypeSeen := make(map[string]bool)
 
 	for i, p := range checkouts {
+		// First we verify the plugin actually implements the hook
+		hookPath, err := b.findHookFile(p.HooksDir, name)
+		// os.IsNotExist() doesn't unwrap wrapped errors (as at Go 1.13).
+		// agent is still go pre-1.13 compatible (I think) so we're avoiding errors.Is().
```

Comment:
I'd say we just bump to latest and use the nice things. But that's separate from this PR.